### PR TITLE
26_User_Interface

### PR DESCRIPTION
**Pull request by y-martinez**
## Summary

This PR close y-martinez/PyZombis#26
## Checklist

- [X] Variables, functions and comments are translated to Spanish
- [X] Functions follow underscore notation
- [X] Spell check done & typos fixed
- [X] All python code is PEP8 compliant
- [X] Test coverage with Playwright implemented; locators are Pyhton code
- [X] Reviewers assigned (all peers & at least 1 mentor)

### CODE REVIEW

File: _sources/lectures/TWP50.rst [hunk 1]
```diff
@@ -4,335 +4,364 @@ Interfaz gráfica
...
-.. image:: img/TWP50_001.png
-   :height: 12.571cm
-   :width: 7.165cm
-   :align: center
-   :alt: 
+    from browser import document, html
 
+    url_audio = "" # Agrega un enlace de un audio para reproducirlo
+    print("Creando el elemento audio")
+    document <= html.AUDIO(id="audio" src=url_audio, controls=True)
```
Comment (angelasofiaremolinagutierrez) [inline _sources/lectures/TWP50.rst:52]: A `,` is missing this generates an error
```suggestion
    document <= html.AUDIO(id="audio", src=url_audio, controls=True)
```

File: _sources/lectures/TWP50.rst [hunk 2]
```diff
@@ -4,335 +4,364 @@ Interfaz gráfica
...
+    from browser import document, html
+    from browser.widgets.dialog import Dialog
 
-   b2 = Button(app,text = "Mal!", width = 10)
-   b2.pack(side = "right",padx = 10,pady = 10)
+    # Creación de un dialogo con el título
+    d = Dialog("Show de preguntas Zombis")
 
+    # Agrega elementos como botones y títulos
+    d.panel <= html.H3(Aprieta los botones, id="titulo_ventana", style=dict(textAlign="center"))
```
Comment (angelasofiaremolinagutierrez) [inline _sources/lectures/TWP50.rst:182]: This also generates an error
```suggestion
    d.panel <= html.H3("Aprieta los botones", id="titulo_ventana", style=dict(textAlign="center"))
```

**Review by angelasofiaremolinagutierrez — CHANGES_REQUESTED** [review]
This looks great! I really like it 👍  However, there is two exercises that have an error within them.

**Comment by y-martinez** [comment]
> This looks great! I really like it  However, there is two exercises that have an error within them.

Ohh gosh I forgot, I was thinking that I already had been resolved them. Thanks for pointing them I am going to change them

**Review by angelasofiaremolinagutierrez — APPROVED** [review]
Amazing! 💯

File: _sources/lectures/TWP50.rst [hunk 3]
```diff
@@ -4,335 +4,364 @@ Interfaz gráfica
...
+    from browser import document, html
+    from browser.widgets.dialog import Dialog
 
-   from tkinter import *
+    d = Dialog("Prueba capturando un evento")
 
-   app = Tk()
-   app.title("Show de primer año")
-   app.geometry('300x100+200+100')
+    d.panel <= html.BUTTON("¡Aprietame!", id="btn_apretar", style=dict(margin="auto", display="block"))
```
Comment (LeoCumpli21) [inline _sources/lectures/TWP50.rst:210]:
```suggestion
    d.panel <= html.BUTTON("¡Apriétame!", id="btn_apretar", style=dict(margin="auto", display="block"))
```

**Review by LeoCumpli21 — APPROVED** [review]
Looks incredible! Congrats.

**Review by NicolasSandoval — APPROVED** [review]
Great job! Thanks.

**Pull request merged**